src/features/accounts/components/accounts-menu/accounts-menu.tsx

```diff
@@ -66,7 +68,25 @@ export function AccountsMenu() {
     })
   );
 
-  const [editAccount, setEditAccount] = React.useState<
+  const toast = useToast();
+  const { services } = useNeighborhoodContext();
+  useEffect(() => {
+    (async () => {
+      const isWebAuthnIdentity =
+        activeAccount?.identity instanceof WebAuthnIdentity;
+      if (isWebAuthnIdentity && !services.has("idstore")) {
```

Comment:
Getting the list of services is no longer asynchronous, which I suspect was causing a problem here.